### ISSUE TITLE
[BUG] Source path for config_files in 'rabbitmq' tree only

### ISSUE THREAD

**Issue by daks** [saltstack-formulas/rabbitmq-formula#56]
### Formula commit hash / release tag

0.17.0

### Versions reports (master & minion)

Salt 2019.2.0 on Debian Stretch mostly

### Pillar / config used

```
rabbitmq:
  config_files:
    rabbitmq.conf:
      source: custom_files/rabbitmq/rabbitmq.conf
```

---

### Describe the bug

The formula let you push any config file for Rabbitmq using the pillar `rabbitmq:config_files` and the state `rabbitmq.config_files`, but the code used is https://github.com/saltstack-formulas/rabbitmq-formula/blob/master/rabbitmq/config_files.sls#L5

```
{% for filename, info in salt["pillar.get"]("rabbitmq:config_files", {}).items() %}
/etc/rabbitmq/{{ filename }}:
  file.managed:
    - source: salt://{{ slspath }}/{{ info['source'] }}
    - template: jinja
    - context: {{ info.get('context', {})|json }}
    - watch_in:
      - service: rabbitmq-server
{% endfor %}
```

so the source of the config file must be in `slspath` (i.e: `rabbitmq`). This is not very flexible depending of how you install/use formulas.

### Steps to reproduce the bug

* create a config file for rabbitmq
* put it outside the formula path
* declare it in your pillar

### Expected behaviour

I expect to be able to put my config file whatever it better suits my organization.

### Attempts to fix the bug

I started coding a quick fix to this problem, but I'm not sure if another solution shouldn't better: TOFS maybe. As I don't really know/use it, I can't judge. But before implementing another solution, if TOFS is the way to go, maybe it should be better to implement it this way. In this case, I'll let another one do it.

**Comment by myii** [comment]
@daks Could you give some example `salt://` paths that you might use?  Then we can determine if TOFS fits the bill.  I'd be surprised if it didn't, though.

**Comment by daks** [comment]
I just want to be able to put my rabbitmq files wherever I can. In my actual use-case, we have "service" wrapping the use of rabbitmq, so I would like to put it in `services/<my-service>/files/rabbitmq.conf` but it's just an example.

if TOFS is not the actual solution, I'll just propose PR for a more 'standard' way to provide via pillar the actual path, something like a 'source' parameter.

**Comment by myii** [comment]
@daks Having thought about this, I'm not sure this is the right time to introduce TOFS since that will introduce a breaking change (the current `source` won't be left in the list).  So if you are going to proceed with your PR, could you please leave the current `source` in place, alongside the new `source` you want to introduce (i.e. using a list)?

**Comment by daks** [comment]
OK, I'll try to do that, with no breaking change.

**Comment by myii** [comment]
Fixed by #57.
